rapid7_attackerkb/icon_rapid7_attackerkb/util/api.py

```diff
@@ -0,0 +1,47 @@
+from requests import request
+
+import komand
+from komand.exceptions import PluginException
+
+
+class AttackerKB:
+    def __init__(self, token, logger, max_pages):
+        self.logger = logger
+        self.token = token
+        self.base_url = 'https://api.attackerkb.com/'
+        if max_pages:
+            self.max_pages = max_pages
+        else:
+            self.max_pages = 100
+
+    def call_api(self, path, params=None):
+        if params is None:
+            params = {}
+
+        api_url = self.base_url + path
+
+        headers = {
+            'Authorization': f'basic {self.token}'
```

Comment:
Keep quotation style consistent (single vs double) throughout the project please